src/pages/EditMonitor.vue

```diff
@@ -489,6 +539,54 @@
                                 </button>
                             </div>
 
+                            <!-- Kafka SASL Options -->
+                            <!-- Kafka Producer only -->
+                            <template v-if="monitor.type === 'kafka-producer'">
+                                <h2 class="mt-5 mb-2">{{ $t("Kafka SASL Options") }}</h2>
+                                <div class="my-3">
+                                    <label class="form-label" for="kafkaProducerSaslMechanism">
+                                        {{ $t("Mechanism") }}
+                                    </label>
+                                    <VueMultiselect
+                                        id="kafkaProducerSaslMechanism"
+                                        v-model="monitor.kafkaProducerSaslOptions.mechanism"
+                                        :options="kafkaSaslMechanismOptions"
+                                        :multiple="false"
+                                        :close-on-select="true"
+                                        :clear-on-select="false"
+                                        :preserve-search="false"
+                                        :placeholder="$t('Pick a SASL Mechanism...')"
```

Comment:
Please add all newly translated strings to the `en.json` translation file. (and others if you are a native speaker)
Otherwise they won't show up in weblate and our translation community cannot localise this ^^

Reply:
I will do it before making this to ready to review

Since I'm into self testing phase

Reply:
Added, I'm Persian native speaker, but I'm not good at translating these things.!